README.md

```diff
@@ -18,7 +18,7 @@ const puppeteer = require('puppeteer')
 const {
   getDevtoolsPanel,
   setCaptureContentScriptExecutionContexts,
-  getContentScriptExcecutionContext
+  getContentScriptExecutionContext
```

Comment:
Just a very minor nit: can we make it a non breaking change for now and have the old name aliased to the correct name?